includes/TripalImporter/BibtexImporter.inc

```diff
@@ -73,6 +69,7 @@ class BibtexImporter extends TripalImporter {
 
 
     $form['instructions'] = [
+    '#weight' => 100,
```

Comment:
Just a quick note: the `-100` (note the negative) is important since weights in Drupal go be "lightness" so the smallest number floats to the top ;-)